src/main/java/frc/robot/subsystems/ROS.java

```diff
@@ -60,15 +63,17 @@ public static void updateTables() {
     portEncoderEntry.setDouble(Drivetrain.getDriveEncoderPort());
     rosIndex.setNumber(rosIntex);
 
-    // Increase the Index value
+    networkTableInstance.flush(); // Force an update and flush all values out.
```

Comment:
suggest also putting a comment here with a link to the chief Delphi post recommending it (for posterity)